### ISSUE TITLE
upload OpsMan Image fails on GCP

### ISSUE THREAD

**Issue by mcb** [pivotal-cf/pcf-pipelines#75]
Tried out this project as you deprecated the older gcp-concourse project. However when using the pipeline running the `create-infrastructure` bit, this fails not finding the terraform command. 

```pcf-pipelines/tasks/install-pcf-gcp/upload-opsman.sh: line 30: terraform: command not found```

**Comment by krishicks** [comment]
@mcb I pushed an update to this task. Let me know how it goes.

**Comment by mcb** [comment]
@krishicks That did the job.

Going forward, I discovered that the google API does not like the description field, which I removed in my fork.

```
Errors:

  * google_compute_image.ops-mgr: "description": this field cannot be set
```

As you switched to Terraform, you might want to adjust the `gs://` Path as this doesn't work and returns another Error: 

```
Errors:

  * google_compute_image.ops-mgr: Error creating image: googleapi: Error 400: Invalid value for field 'resource.id': '0'. The provided source location was not a valid Google Cloud Storage object reference., invalid
```

This can be solved by using the Storage Path `https://storage.googleapis.com/` instead of `gs://`.

Once this is uploaded, Terraform doesn't check whether it is already uploaded on a second run. Maybe we might want to trigger a check before as we wouldn't need to upload it again. The old pipeline had that check built in.
